services/logger.js

```diff
@@ -0,0 +1,109 @@
+const createLogger = () => {
+  const isProdMode =
+    process.env.NODE_ENV === "prod" || process.env.NODE_ENV !== "development"
+      ? true
+      : false;
+
+  // level "debug" = debug
+  // level "default" = default
+  // level "none" = no logging
+  //TODO: move this var to env file.
+  let logLevel = "default";
+
+  if (isProdMode) {
+    return createProdLogger(logLevel);
+  }
+
+  return createDevLogger(logLevel);
+};
+
+const createProdLogger = (logLevel) => {
+  //TODO: read log destination from env. (QA/Prod might be different destination, but log the same)
+  //TODO: implement a real Prod logger
+
+  const debugLogFunction = (message) => {
+    if (logLevel !== "debug") return;
+
+    const currentTime = new Date().toLocaleTimeString();
+    console.log(`DEBUG - ${currentTime}: ${message}`);
```

Comment:
```suggestion
    console.debug(`DEBUG - ${currentTime}: ${message}`);
```
So that you can leverage devtools logs filtering

Reply:
I like it